ASM/c/item_table.h

```diff
@@ -6,6 +6,326 @@
+    /* 0x004A */ GI_ARROWS_10, // Arrows (10)
+    /* 0x004B */ GI_ARROWS_30, // Arrows (30)
+    /* 0x004C */ GI_RUPEE_GREEN, // Green Rupee
+    /* 0x004D */ GI_RUPEE_BLUE, // Blue Rupee
+    /* 0x004E */ GI_RUPEE_RED, // Red Rupee
+    /* 0x004F */ GI_HEART_CONTAINER_2, // Heart Container
+    /* 0x0050 */ GI_MILK, // Lon Lon Milk (Refill)
+    /* 0x0051 */ GI_MASK_GORON, // Goron Mask
+    /* 0x0052 */ GI_MASK_ZORA, // Zora Mask
+    /* 0x0053 */ GI_MASK_GERUDO, // Gerudo Mask
+    /* 0x0054 */ GI_GORONS_BRACELET, // Goron's Bracelet
+    /* 0x0055 */ GI_RUPEE_PURPLE, // Purple Rupee
+    /* 0x0056 */ GI_RUPEE_GOLD, // Huge Rupee
+    /* 0x0057 */ GI_SWORD_BIGGORON, // Biggoron's Sword
+    /* 0x0058 */ GI_ARROW_FIRE, // Fire Arrow
+    /* 0x0059 */ GI_ARROW_ICE, // Ice Arrow
+    /* 0x005A */ GI_ARROW_LIGHT, // Light Arrow
+    /* 0x005B */ GI_SKULL_TOKEN, // Gold Skulltula Token
+    /* 0x005C */ GI_DINS_FIRE, // Din's Fire
+    /* 0x005D */ GI_FARORES_WIND, // Farore's Wind
+    /* 0x005E */ GI_NAYRUS_LOVE, // Nayru's Love
+    /* 0x005F */ GI_BULLET_BAG_30, // Bullet Bag (30)
+    /* 0x0060 */ GI_BULLET_BAG_40, // Bullet Bag (40)
+    /* 0x0061 */ GI_DEKU_STICKS_5, // Deku Sticks (5)
+    /* 0x0062 */ GI_DEKU_STICKS_10, // Deku Sticks (10)
+    /* 0x0063 */ GI_DEKU_NUTS_5_2, // Deku Nuts (5)
+    /* 0x0064 */ GI_DEKU_NUTS_10, // Deku Nuts (10)
+    /* 0x0065 */ GI_BOMBS_1, // Bomb
+    /* 0x0066 */ GI_BOMBS_10, // Bombs (10)
+    /* 0x0067 */ GI_BOMBS_20, // Bombs (20)
+    /* 0x0068 */ GI_BOMBS_30, // Bombs (30)
+    /* 0x0069 */ GI_DEKU_SEEDS_30, // Deku Seeds (30)
+    /* 0x006A */ GI_BOMBCHUS_5, // Bombchu (5)
+    /* 0x006B */ GI_BOMBCHUS_20, // Bombchu (20)
+    /* 0x006C */ GI_BOTTLE_FISH, // Fish (Refill)
+    /* 0x006D */ GI_BOTTLE_BUGS, // Bugs (Refill)
+    /* 0x006E */ GI_BOTTLE_BLUE_FIRE, // Blue Fire (Refill)
+    /* 0x006F */ GI_BOTTLE_POE, // Poe (Refill)
+    /* 0x0070 */ GI_BOTTLE_BIG_POE, // Big Poe (Refill)
+    /* 0x0071 */ GI_DOOR_KEY, // Small Key (Chest Game)
+    /* 0x0072 */ GI_RUPEE_GREEN_LOSE, // Green Rupee (Chest Game)
+    /* 0x0073 */ GI_RUPEE_BLUE_LOSE, // Blue Rupee (Chest Game)
+    /* 0x0074 */ GI_RUPEE_RED_LOSE, // Red Rupee (Chest Game)
+    /* 0x0075 */ GI_RUPEE_PURPLE_LOSE, // Purple Rupee (Chest Game)
+    /* 0x0076 */ GI_HEART_PIECE_WIN, // Piece of Heart (Chest Game)
+    /* 0x0077 */ GI_DEKU_STICK_UPGRADE_20, // Deku Stick Upgrade (20)
+    /* 0x0078 */ GI_DEKU_STICK_UPGRADE_30, // Deku Stick Upgrade (30)
+    /* 0x0079 */ GI_DEKU_NUT_UPGRADE_30, // Deku Nut Upgrade (30)
+    /* 0x007A */ GI_DEKU_NUT_UPGRADE_40, // Deku Nut Upgrade (40)
+    /* 0x007B */ GI_BULLET_BAG_50, // Bullet Bag (50)
+    /* 0x007C */ GI_ICE_TRAP, // Ice Trap
+    /* 0x007D */ GI_TEXT_0,
+    /* 0x007D */ GI_CAPPED_PIECE_OF_HEART = GI_TEXT_0, // Capped Piece of Heart
+    /* 0x007E */ GI_VANILLA_MAX,
+    /* 0x007E */ GI_CAPPED_HEART_CONTAINER = GI_VANILLA_MAX, // Capped Heart Container
```

Comment:
Can you explain this a little bit? I'm confused why the first 2 instances of the same ID are here

Reply:
Those are the vanilla values. GI_TEXT_0 is the pocket egg error id text that rando repurposes and GI_VANILLLA_MAX is where the vanilla list ends.

Reply:
Ah ok, in that case it makes sense to document the original IDs as well as what we're using them for here.